Doc/reference/expressions.rst

```diff
@@ -1874,7 +1878,7 @@ Expression lists
    single: , (comma); expression list
 
 .. productionlist:: python-grammar
-   expression_list: `expression` ("," `expression`)* [","]
+   expression_list: `starred_expression` ("," `starred_expression`)* [","]
```

Comment:
This is not strictly correct. `expression_list` according to the grammar is what appears after `yield`, but `def f(): yield *a` is a SyntaxError. (However, `yield a, *b` is legal.)